bugbug/models/duplicate.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from imblearn.over_sampling import BorderlineSMOTE
+from sklearn.calibration import CalibratedClassifierCV
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import LabelEncoder
+from sklearn.svm import LinearSVC
+
+from bugbug import bugzilla, feature_cleanup
+from bugbug.model import BugCoupleModel
+
+
+class super_classifier(CalibratedClassifierCV):
+    def __init__(self, clf):
+        super().__init__(LinearSVC())
+        self._le = LabelEncoder()
+
+    def fit(self, X, y):
+        super().fit(X, y)
+        self._le.fit(y)
+
+
+class DuplicateModel(BugCoupleModel):
+    def __init__(self, lemmatization=False):
+
+        BugCoupleModel.__init__(self, lemmatization)
+
+        self.cross_validation_enabled = True
+        self.sampler = BorderlineSMOTE(random_state=0)
+        self.calculate_importance = False
+
+        self.bugs = {}
```

Comment:
We can't store the bugs as a property of the model, or the model will be huge.
Also, the dataset of bugs will be different in production.